src/js/background/background.js

```diff
@@ -268,8 +262,6 @@ async function makeRelayAddress(description = null) {
   ]);
   browser.storage.local.set({ relayAddresses: updatedLocalRelayAddresses });
 
-  await refreshAccountPages();
-
```

Comment:
We still need a refresh mechanism to update the dashboard (if open) when the add-on generates a new mask. I want to flag this for a future enhancement.

Reply:
Yeah that sounds good. I don't think we used to have that before either, did we? Otherwise we can just re-use that mechanism (i.e. stick with refreshing until we have proper add-on-website comms).